contrib/fuzz/oss_fuzz_build.sh

```diff
@@ -17,10 +17,47 @@
 cd "$(dirname "${BASH_SOURCE[0]}")"
 cd ../../
 
-# Don't move docker_fuzzer.go back into contrib/fuzz
+# Move all fuzzers that don't have the "fuzz" package out of this dir
 mv contrib/fuzz/docker_fuzzer.go remotes/docker/
-compile_go_fuzzer github.com/containerd/containerd/remotes/docker FuzzFetcher fuzz_fetcher
+mv contrib/fuzz/container_fuzzer.go integration/client/
+
 
+compile_go_fuzzer github.com/containerd/containerd/remotes/docker FuzzFetcher fuzz_fetcher
 compile_go_fuzzer github.com/containerd/containerd/contrib/fuzz FuzzFiltersParse fuzz_filters_parse
 compile_go_fuzzer github.com/containerd/containerd/contrib/fuzz FuzzPlatformsParse fuzz_platforms_parse
 compile_go_fuzzer github.com/containerd/containerd/contrib/fuzz FuzzApply fuzz_apply
+
+
+# FuzzCreateContainer requires more setup than the fuzzers above.
+# We need the binaries from "make".
+wget -c https://github.com/google/protobuf/releases/download/v3.11.4/protoc-3.11.4-linux-x86_64.zip
```

Comment:
There was a break in this redirect earlier today which has since been fixed, but better here to just link to the new location like we did in our other script.
```
wget -c https://github.com/protocolbuffers/protobuf/releases/download/v3.11.4/protoc-3.11.4-linux-x86_64.zip

```